src/main/java/es/unizar/smartcampuz/model/reservation/ReservationChecker.java

```diff
@@ -3,19 +3,35 @@
 
 public class ReservationChecker {
 
-    public static final int START_HOUR = 7;
-    public static final int FINISH_HOUR = 19;
-
-    // TODO: El iterable hay que cambiar ? por una Reservation
-    public static boolean checkSchedule (boolean [] reservation, Iterable<?> iter){
-
-        for(Object o: iter){
-            // TODO: Cogeré este array del objeto Reservation
-            boolean [] reserved = new boolean[24];
-            for(int i=START_HOUR;i<=FINISH_HOUR;i++){
-                if(reservation[i] && reserved[i]){
-                    return false;
-                }
+    //Represents first valid TimeSlot 8:00-9:00
+    public static final int START_TIME_SLOT = 8;
+
+    //Represents last valid TimeSlot 20:00-21:00
+    public static final int FINISH_TIME_SLOT =20;
+    public static final int NUM_TIME_SLOTS = 24;
```

Comment:
This constant is redundant with the one in "TimeReservation", and that could be dangerous. It would be better to take it from that class, since you only use this constant here ;)